pulp_puppet_plugins/pulp_puppet/plugins/distributors/installdistributor.py

```diff
@@ -216,7 +216,8 @@ def _rename_directory(unit, destination, names):
 
         before = os.path.join(destination, dir_names.pop())
         after = os.path.join(destination, unit.unit_key['name'])
-        shutil.move(before, after)
+        if not os.path.samefile(before, after):
```

Comment:
Will ``after`` always exist? If it does not exist, this call will raise an ``OSError``.